Makefile

```diff
@@ -74,30 +74,16 @@ clean:
 generatecode:
 	mkdir -p $(OUTPUT_DIR)
 	go run cmd/generate-code/main.go
-
-# Docs
-.PHONY: generatedocs
-generatedocs:
```

Comment:
This wasn't getting run, following the other projects we run generated-docs when we run generate-code